packages/expo-intent-launcher/android/src/main/java/expo/modules/intentlauncher/IntentLauncherModule.java

```diff
@@ -68,8 +68,15 @@ public void startActivity(@NonNull String activityAction, @NonNull ReadableArgum
       promise.reject(new ModuleNotFoundException("UIManager"));
       return;
     }
+    
+    Intent intent = null;
+    
+    if (activityAction != null) {
+      intent = new Intent(activityAction);
+    } else { 
+      intent = new Intent();
+    }
```

Comment:
Looks like we don't even need this if..else, `Intent intent = new Intent(activityAction);` should work as expected if `activityAction` is `null`. So, removing `@NonNull` is the only thing we need to change here to fix it 😉